scripts/scil_sh_to_aodf.py

```diff
@@ -55,39 +54,58 @@ def _build_arg_parser():
 
     add_sh_basis_args(p)
 
-    p.add_argument('--sphere', default='repulsion724',
+    p.add_argument('--sphere', default='repulsion200',
```

Comment:
Out of curiosity, why use such a low resolution sphere?

Reply:
For faster execution. It's the value I use in the paper and it did not appear to make much difference when compared to a sphere of higher resolution.